### DISCUSSION TITLE
[Support]: Bug on saving camera groups

### DISCUSSION THREAD

**Discussion by sylv3rblade** [blakeblackshear/frigate#11345] [General Support] [answered]
### Describe the problem you are having

UI bug for 0.14 dev, this occurs when saving a camera group.

You get this weird behavior if the content is long enough to present a scrollbar

https://github.com/blakeblackshear/frigate/assets/467007/f5354e64-bc43-415b-b873-24506717efac

### Version

0.14.0-021ffb24

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Frigate stats

```json
n/a
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

Tapo 325WB

**Comment by hawkeye217** [comment, marked as answer]
This was fixed in https://github.com/blakeblackshear/frigate/pull/11334. If you update to the latest dev image and still have the issue, let us know!

**Reply by sylv3rblade** [reply]
ah shoot turns out i built the new image from dev but forgot to update the compose file so it wasnt getting used, my bad.  Thank you, will close,
